app/src/main/java/com/kickstarter/libs/DeviceRegistrar.java

```diff
@@ -29,7 +31,13 @@ public void registerDevice() {
       return;
     }
 
-    this.context.startService(new Intent(this.context, RegisterService.class));
+    final FirebaseJobDispatcher dispatcher = new FirebaseJobDispatcher(new GooglePlayDriver(this.context));
```

Comment:
Looks like we're doing the same thing 3 times, which seems like a Util class would be helpful! Let's make a Util class for Jobs™. So you'd pass in a context, the class, and the tag. And then we could make these tags constants!